bpf/lib/lb.h

```diff
@@ -29,9 +29,9 @@ struct bpf_elf_map __section_maps LB6_SERVICES_MAP_V2 = {
 	.flags		= CONDITIONAL_PREALLOC,
 };
 
-struct bpf_elf_map __section_maps LB6_BACKEND_MAP = {
+struct bpf_elf_map __section_maps LB6_BACKEND_MAP_V2 = {
```

Comment:
True, but maybe we want to keep V2 in its name for better readability (in case we find a bug in the future due to this map key type change)? I see service map has a V2 in its name too :)